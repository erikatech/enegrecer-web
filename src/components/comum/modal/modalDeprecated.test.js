import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Modal from './modalDeprecated';

describe('Modal', () => {
  it('deve construir o componente corretamente ', () => {
    const modal = shallow(<Modal />);
    const modalJson = toJson(modal);
    expect(modalJson).toMatchSnapshot();
  });
});
