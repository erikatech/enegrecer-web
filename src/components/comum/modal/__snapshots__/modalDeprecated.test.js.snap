// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal deve construir o componente corretamente  1`] = `
<div
  className="modal"
  id=""
>
  <div
    className="modal-content"
  >
    <h4 />
    <p />
  </div>
  <div
    className="modal-footer"
  >
    <a
      className="modal-action modal-close waves-effect waves-green btn-flat"
      href="#!"
    />
  </div>
</div>
`;
